setup.cfg

```diff
@@ -33,18 +33,8 @@ target-version = ['py37']
 ignore=
     # check ignored by default in flake8. Meaning unclear.
     E24,
-    # continuation line under-indented
-    E121,
-    # closing bracket does not match indentation
-    E123,
-    # continuation line over-indented for hanging indent
-    E126,
     # space before : (needed for how black formats slicing)
     E203,
-    # missing whitespace around arithmetic operator
-    E226,
-    # multiple statements on one line (def)
-    E704,
```

Comment:
All these error codes seem outdated since we are black compliant. I check removing them and we indeed don't violate any of them. I think it's a good opportunity to do some clean-up